tests/unit/test_dojson_utils.py

```diff
@@ -40,82 +38,6 @@
 )
 
 
-def test_classify_field_returns_none_on_falsy_value():
-    assert classify_field('') is None
-
-
-def test_classify_field_returns_none_on_non_string_value():
-    assert classify_field(0) is None
-
-
-def test_classify_field_returns_category_if_found_among_keys():
-    expected = 'Math and Math Physics'
-    result = classify_field('alg-geom')
-
-    assert expected == result
-
-
-def test_classify_field_returns_category_if_found_among_values():
-    expected = 'Astrophysics'
-    result = classify_field('Astrophysics')
-
-    assert expected == result
-
-
-def test_classify_field_ignores_case():
-    expected = 'Astrophysics'
-    result = classify_field('ASTRO-PH.CO')
-
-    assert expected == result
-
-
-def test_classify_field_returns_none_on_unknown_values():
-    assert classify_field('FOO') is None
-
-
-def test_classify_rank_returns_none_on_falsy_value():
-    assert classify_rank('') is None
```

Comment:
Even if now `classify_rank` is completely trivial, preserving its tests would ensure that the new implementation is not regressing / introducing different behavior.